plugins/catalog-backend/src/database/DefaultProcessingDatabase.ts

```diff
@@ -158,7 +159,7 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
       'refresh_keys',
       refreshKeys.map(k => ({
         entity_id: id,
-        key: k.key,
+        key: createHash('sha256').update(k.key).digest('hex'),
```

Comment:
Hm. Would it make sense to make this such that if the length of the string is greater than the limit, ONLY then hash it? This would (1) increase performance, and (2) make the database contents generally much more debuggable / understandable for the majority use case.

Make a (tested) utility function for this, which can be used in all places that need it.

Best of all would probably be to make the hashed value be `<the first 150 characters of the string>` + `#sha256:` + `<the hash of the entire string>` or whatever, so that you can at least see a glimpse of what it's pointing to, and understand what's going on